pixeltable/client.py

```diff
@@ -210,7 +212,7 @@ def create_view(
         _logger.info(f'Created view {path_str}')
         return view
 
-    def get_table(self, path: str) -> catalog.Table:
+    def get_table(self, path: str) -> catalog.SchemaObject:
```

Comment:
The typechecker doesn't know it's a Table, because it's fetched from a PathDict which returns a SchemaObject.

But I agree it should be a Table and probably the best pattern in this case is
```
obj = self.catalog.paths[p]
assert isinstance(obj, Table)
return obj
```

I'll fix it.